README.md

```diff
@@ -107,6 +107,7 @@ This extension provides below configurations in settings.json.
 |config name| description|
 |--|--|
 |`ansible.hover`| Enable/Disable hover over module name functionality. Default is true.|
+|`ansible.autocompletion` | Enable/Disable ansible autocompletion(including code snippets) functionality. Default is true. To enable ansible autocompletion only in specific yaml files, set `ansible.autocompletion = false`, then add `# ansible-configured` header in first line of yaml file.|
```

Comment:
How about adding a new column for `Default Value`. So that it is easier for user to find out the default value.